python/plugins/db_manager/db_plugins/spatialite/plugin.py

```diff
@@ -175,6 +175,9 @@ def explicitSpatialIndex(self):
     def spatialIndexClause(self, src_table, src_column, dest_table, dest_column):
         return u""" "%s".ROWID IN (\nSELECT ROWID FROM SpatialIndex WHERE f_table_name='%s' AND search_frame="%s"."%s") """ % (src_table, src_table, dest_table, dest_column)
 
+    def supportsComment(self):
```

Comment:
Move this method `getComment` into the base class `DBConnector` as a default implementation and override in PG connector only, instead of providing the same default implementation in all connectors but the PG one.